Tasks/AzureRmWebAppDeployment/Tests/L0WindowsFailSetParamFile.ts

```diff
@@ -132,7 +132,18 @@ tr.registerMock('webdeployment-common/azurerestutility.js', {
 		}
 
 		return config;
-	}
+	},
+    updateDeploymentStatus: function(publishingProfile, isDeploymentSuccess ) {
```

Comment:
Remove this duplicate code and put it a common place and return that in both the mocks